Lib/test/support/__init__.py

```diff
@@ -3040,6 +3040,19 @@ class catch_unraisable_exception:
     """
     Context manager catching unraisable exception using sys.unraisablehook.
 
+    The the *object* attribute of the unraisable hook is set and the object is
+    being finalized, the object is resurected because the context manager
```

Comment:
```suggestion
    being finalized, the object is resurrected because the context manager
```

resurected -> resurrected in below content too.